src/components/access_group/AccessGroupCreate.vue

```diff
@@ -147,6 +248,7 @@ export default {
         name: this.groupName,
         type: this.groupType,
         description: this.groupDescription,
+        group_expiration: this.selectedExpiration,
```

Comment:
```suggestion
        group_expiration: Number(this.selectedExpiration),
```

Reply:
I don't understand where this suggestion comes from. It already sends the group_expiration as a number.